node/pkg/admin/adapter/queries.go

```diff
@@ -0,0 +1,36 @@
+package adapter
+
+const (
+	InsertAdapter = `INSERT INTO adapters (name) VALUES (@name) RETURNING *;`
+
+	InsertFeed = `INSERT INTO feeds (name, definition, adapter_id) VALUES (@name, @definition, @adapter_id) RETURNING *;`
+
+	GetAdapter = `SELECT * FROM adapters;`
+
+	GetAdapterById = `SELECT * FROM adapters WHERE adapter_id = @id;`
+
+	GetFeedsByAdapterId = `SELECT * FROM feeds WHERE adapter_id = @id;`
+
+	DeleteAdapterById = `DELETE FROM adapters WHERE adapter_id = @id;`
+)
+
+// try later
```

Comment:
How about removing it and making it as an issue? 🤔

Reply:
Yes I'll make it as an issue and remove comments